### ISSUE TITLE
Can't use faunadb-go

### ISSUE THREAD

**Issue by StarpTech** [fauna/faunadb-go#87]
Hello,
I want to evaluate FaunaDB but it's not possible due to https://github.com/fauna/faunadb-go/issues/77

**Error:**

```
go get github.com/fauna/faunadb-go@v2.11.0: github.com/fauna/faunadb-go@v2.11.0: invalid version: module contains a go.mod file, so major version must be compatible: should be v0 or v1, not v2
```

Therefore the CI wasn't fixed for more than 20 days. Is this module still maintained?

Related: https://github.com/golang/go/issues/35732

**Comment by trevorsibanda** [comment]
Hi @StarpTech 

You can retrieve the latest faunadb-go driver by using `go get github.com/fauna/faunadb-go` fetching a specific version isn't currently possible with `go get` without specifiying the commit hash available from the releases tab.

i.e `go get github.com/fauna/faunadb-go@ffe381025f9bc91945be8a7cf6de3bbb27ab16ea` for v2.11.0
`go get github.com/fauna/faunadb-go@ad1b5145dca242f7dd377191de8a74a15fbee62e` for v2.10.0

See #84

**Comment by StarpTech** [comment]
`go get github.com/fauna/faunadb-go` doesn't work it will fetch `github.com/fauna/faunadb-go v2.0.0+incompatible`

I also tried `go get github.com/fauna/faunadb-go@ffe381025f9bc91945be8a7cf6de3bbb27ab16` but it also doesn't work.

```
imports
        github.com/fauna/faunadb-go: module github.com/fauna/faunadb-go@latest found (v2.0.0+incompatible), but does not contain package github.com/fauna/faunadb-go
```

What's the import path?

**Comment by StarpTech** [comment]
I tried `f "github.com/fauna/faunadb-go/faunadb"` but `f.Collection()` is not available in any of the examples.

**Comment by nathaniel-may** [comment]
Hi @StarpTech, thank you for your patience.

We're currently working on adding support for go modules so the `path@version` syntax isn't available yet.

Could you try installing the driver using: 
`go get gopkg.in/fauna/faunadb-go.v2/faunadb`

And changing the import in your code to use the same path?

**Comment by StarpTech** [comment]
Hi @nathaniel-may here is a repl.

https://repl.it/@StarpTech/UnderstatedSpatialSysadmin

Output:

```
go version go1.13 linux/amd64
 go build
go: downloading github.com/fauna/faunadb-go v2.0.0+incompatible
# main
./main.go:16:47: undefined: faunadb.Collection
compiler exit status 2
```

**Comment by nathaniel-may** [comment]
Thanks! The script you linked to looks good, but from the output it looks like it's still attempting to download the driver from the github path.

Before building could you check that the driver exists in the following path?
`$GOPATH/pkg/<your-arch>/gopkg.in/fauna/faunadb-go.v2`

If it doesn't exist there yet, running the following command should install it there.
`go get gopkg.in/fauna/faunadb-go.v2/faunadb`

**Comment by StarpTech** [comment]
> Thanks! The script you linked to looks good, but from the output it looks like it's still attempting to download the driver from the github path.

The message describes that the package does not export the `Collection` function.
`go vet` returns `vet: ./main.go:43:46: Collection not declared by package faunadb`

> Before building could you check that the driver exists in the following path?
$GOPATH/pkg/mod/gopkg.in/fauna/faunadb-go.v2

I think you mean `$GOPATH/pkg/mod/gopkg.in/fauna/faunadb-go.v2@v2.0.0` ? I use go modules. Yes, it is there and it's **not** the latest version.

Did you try it on your own?

**Comment by nathaniel-may** [comment]
We don't support go modules at this time but we will in the near future.

**Comment by StarpTech** [comment]
Thanks, this explains a lot.

**Comment by nathaniel-may** [comment]
Closing this issue. Feel free to open another if you need anything else.

**Comment by kainosnoema** [comment]
Also needing go modules support. Modules are fast becoming the standard way of managing dependencies—any news on when this will be available?

**Comment by camilopayan** [comment]
I believe this is resolved by #112
